auth/deployment.yaml

```diff
@@ -81,10 +81,15 @@ spec:
                 key: gsuite_organization
          resources:
            requests:
-             memory: "250M"
-             cpu: "100m"
+{% if deploy %}
+             memory: 375Mi
+             cpu: 100m
+{% else %}
+             memory: 37Mi
```

Comment:
Looking at `k top pods`, the current batch-driver usage is 51Mi (but auth is 37Mi).  Can we leave in a little slack?  Maybe make it 100Mi for auth-driver and 50Mi for auth?

Reply:
Sure. I set batch, batch-driver, auth, and auth-driver, in non-deploy-mode, to 20m / 75Mi. (I'm aiming for a set cpu:mem ratio so as to simplify how I think about resources)